README.md

```diff
@@ -20,6 +20,16 @@ Please use the [ISLE Documentation](https://islandora-collaboration-group.github
 * Time required < 30 minutes.
 * **Windows Users**: Please open the .env and uncomment `COMPOSE_CONVERT_WINDOWS_PATHS=1`
 
+## **AS OF THE ISLE v.1.4.0 release in November 2019 (MANDATORY)**
```

Comment:
Will a PR from a developer using an uncleaned forked repo cause the files to return?  If so, we need to put a clear message in here and in developer documents that we will not accept PRs from repos that haven't had the script run on them.

Reply:
Yes a PR from a developer using an uncleaned forked repo **will indeed** cause the files to return. I'll try to find an appropriate document to add this message in.

Reply:
Okay I've added this message for now in both the README.md and the isle-v140-git-cleanup.md doc file for now.
```- If you intend to submit pull requests (PRs) of any kind (documentation, code etc) to the ISLE project, ISLE maintainers will **NOT** accept PRs from repos that haven't had the script run on them as using an uncleaned forked repo will cause the removed files to return.```

Reply:
Thanks, Gavin!